Creational/Builder/Director.php

```diff
@@ -20,9 +20,12 @@ class Director
     public function build(BuilderInterface $builder)
     {
         $builder->createVehicle();
-        $builder->addDoors();
         $builder->addEngine();
         $builder->addWheel();
+        
+        if ($builder instanceof CarBuilder) {
```

Comment:
Thanks for thinking into consideration my advice. It is great pleasure for me to cooperate with you.
:+1: